### ISSUE TITLE
Working on multiple screens

### ISSUE THREAD

**Issue by omar-3** [marcosfad/mbp-ubuntu#77]
The issue is simply that Ubuntu doesn't acknowledge that the MacBook is connected to an external monitor.

**Comment by AdityaGarg8** [comment]
Is it a thunderbolt display or HDMI.

**Comment by omar-3** [comment]
> Is it a thunderbolt display or HDMI.

HDMI

**Comment by AdityaGarg8** [comment]
I actually don't use external monitors but many users on the discord have got it working. You probably could join the server and ask this over there?

**Comment by omar-3** [comment]
> I actually don't use external monitors but many users on the discord have got it working. You probably could join the server and ask this over there?

Okay, will close the issue for now.

**Comment by omar-3** [comment]
> I actually don't use external monitors but many users on the discord have got it working. You probably could join the server and ask this over there?

Okay, will close the issue for now

**Comment by AdityaGarg8** [comment]
I'll repoen it unless we get a solution, so that we can have a record of the same.

**Comment by lemmyg** [comment]
Hello,
I use an external monitor with my Macbook  pro 16,1 every day. 
Is connected with HDMI to my usb dock. I have beeen using it in Kubuntu and Ubuntu 22.04, 22.10.
I only had one issue with the laptop HiDPI  screen. where in X11 is not possible to scale each screen separately.
The workaround I found was to set both to 100% and reduce the laptop screen from 3070x1920 to 1920x1200.
Also using 100% solved many scale issues I found as some applications may not supported this feature properly.
Note that Im using amdgpu but intel gpu was working too.
I hope this helps.
Regards
